Speckle_Engine/Convert/ToBHoM.cs

```diff
@@ -8,17 +10,56 @@ namespace BH.Engine.Speckle
 {
   public static partial class Convert
   {
-    /***************************************************/
-    /**** Public Methods                            ****/
-    /***************************************************/
-
-    //Add methods for converting to BHoM from the specific software types, if possible to do without any BHoM calls
-    //Example:
-    //public static Node ToBHoM(this SpeckleNode node)
-    //{
-    //    //Insert code for convertion
-    //}
-
-    /***************************************************/
-  }
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static IEnumerable<IBHoMObject> ResponseToBHoM(ResponseObject response, bool setAssignedId = true, List<string> speckleIds = null)
+        {
+            List<IBHoMObject> bhomObjects = new List<IBHoMObject>();
+
+            ResponseToBHoM(response, out bhomObjects, out _, out _, setAssignedId, speckleIds);
+
+            return bhomObjects;
+        }
+
+        public static bool ResponseToBHoM(ResponseObject response, out List<IBHoMObject> bHoMObjects, out List<IObject> iObjects, out List<object> reminder, bool assignSpeckleIdToBHoMObjects = true, List<string> speckleIds = null)
```

Comment:
This is not a `Convert` as it is returning a `bool`. I would either move this into a `Query` method and refactor it in a way to minimise the use of `out` parameters. Probably returning `IObject` and filtering them elsewhere, or returning a `Dictionary<Type, IObject>` may help.